README.md

```diff
@@ -0,0 +1,11 @@
+# Homeboy
+
+Wii VC emulator support for [gz](https://github.com/glankk/gz) and [kz](https://github.com/krimtonz/kz).
+
+## Prerequisites
+
+You will need ``powerpc-eabi-gcc`` (provided by [devkitPro](https://devkitpro.org/wiki/devkitPro_pacman) or [wii-toolchain](https://github.com/krimtonz/wii-toolchain)) to build this project.
```

Comment:
why double backtick? I don't think that's standard markdown

Reply:
Windows is a wonderful OS